### PR TITLE
Explicitly declaring css height for snippet icons.

### PR DESCRIPTION
**Pull request by glogiotatidis**
Don't allow SVG icons get larger than 40x50px by explicitly declaring
their height.

### CODE REVIEW

File: snippets/base/templates/base/includes/snippet_css.html [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
-  .snippet .icon {
-    position: absolute;
-    right: 400px;
-    top: 5px;
-  }
+   .snippet .icon {
+       position: absolute;
+       right: 400px;
+       top: 5px;
+       height: 50px;
```
Comment (glogiotatidis) [inline snippets/base/templates/base/includes/snippet_css.html]: This is the only non-spacing change in this file.

**Comment by mozilla-travis** [comment]
```
snippets/base/models.py:329:101: E501 line too long (101 > 100 characters)

```

**Comment by mozilla-travis** [comment]
```
snippets/base/models.py:329:101: E501 line too long (101 > 100 characters)

```

**Comment by glogiotatidis** [comment]
@Osmose r? I don't have a Retina display so I can't test whether setting px breaks anything on Retina but since we use px in other places I guess it will be fine. I'm curious though how Firefox handles Retina displays? Does it scale everything to 110% for example? Is there a point to using SVGs?

**Comment by Osmose** [comment]
Gotta fix that broken test, but r+ otherwise. Nice work!

I tested on a Retina MacBook and I believe it works in the correct way with this fix:

- If you use an SVG, it is constrained and smooth on a Retina screen.
- If you use a PNG at 40x50, it is constrained but blurry on a Retina screen.
- If you use a PNG at 80x100, it is constrained and smooth on a Retina screen.

So I think you're good!

**Comment by glogiotatidis** [comment]
thanks @Osmose  :)

**Pull request merged**